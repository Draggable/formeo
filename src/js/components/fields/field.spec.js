import Field from './field'

describe('Field', () => {
  it('should match snapshot', () => {
    const fieldConfig = { id: 'test-id' }
    const field = new Field(fieldConfig)
    expect(field).toMatchSnapshot()
  })
})
