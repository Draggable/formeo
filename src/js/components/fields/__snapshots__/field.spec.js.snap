// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Field should have data property matching snapshot 1`] = `
Object {
  "conditions": Array [
    Object {
      "if": Array [
        Object {
          "comparison": "",
          "source": "",
          "sourceProperty": "",
          "target": "",
          "targetProperty": "",
        },
      ],
      "then": Array [
        Object {
          "assignment": "",
          "target": "",
          "targetProperty": "",
          "value": "",
        },
      ],
    },
  ],
  "config": Object {},
  "id": "test-id",
}
`;
